src/drivers/differential_pressure/asp5033/ASP5033.hpp

```diff
@@ -107,8 +107,6 @@ class ASP5033 : public device::I2C, public I2CSPIDriver<ASP5033>
 	float _pressure_prev = 0.f;
 	float _temperaute_prev = 0.f;
 
-	int probe() override;
```

Comment:
So there is no way to do a probe to check it's actually there?

Reply:
Yes, no, maybe?

Reply:
I think it can be checked. Now I tried new checking model.